synapse/lib/storm.py

```diff
@@ -947,3 +948,65 @@ def getArgParser(self):
 
             if self.opts.join:
                 yield node, path
+
+class ScrapeCmd(Cmd):
+    '''
+    Use textual properties of existing nodes to find other easily recognizable nodes.
+
+    Examples:
+
+        inet:search:query | scrape
+        inet:search:query | scrape --refs
+    '''
+
+    name = 'scrape'
+
+    def getArgParser(self):
+        pars = Cmd.getArgParser(self)
+
+        pars.add_argument('--props', '-p', nargs='+', type=str, default=[],
+                          help='Specify relative properties to scrape')
```

Comment:
We need better language here.  We typically see relative properties denoted with `:propname` and we're not actually handling a property given in that manner currently.